webapp/sql/create_pois_table.sql

```diff
@@ -2,14 +2,14 @@ DROP TABLE IF EXISTS pois;
 
 CREATE TABLE pois (
     id SERIAL PRIMARY KEY,
-    geometry geometry,
+    geometry geometry
 );
 
--- Note that building filters some pois such as sports_centre because sports 
--- centre is not a building
-CREATE OR REPLACE VIEW building AS
+-- Note that public_building filters some pois such as sports_centre because 
+-- sports centre is not a public_building
+CREATE OR REPLACE VIEW public_building AS
```

Comment:
Can you make that `nonprivate_building`? While these buildings are (probably) publicly accessible, "public building" can have a narrower meaning than that.